### DISCUSSION TITLE
Can't see preview

### DISCUSSION THREAD

**Discussion by stephaniehanor** [thegetty/quire#205] [Get Help] [answered]
I have installed Quire on a new Mac desktop but I'm unable to see the preview and I get the following error message. I saw that this came up in a previous discussion and the solution was to try quire install and then quire preview, which I tried but I'm still getting this error. I also saw another recommendation for using the LTS version of Node.js, which I have done and is v16.13.0.

Any suggestions? Thanks so much!

mac-974:unseen shanor$ quire preview
Launching preview server
ℹ Navigate to http://localhost:1313 to see your changes.
ℹ Press Control+C or type "quire stop" to stop the preview.
node:events:368
      throw er; // Unhandled 'error' event
      ^

Error: spawn node_modules/.bin/webpack ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (node:internal/child_process:288:12)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn node_modules/.bin/webpack',
  path: 'node_modules/.bin/webpack',
  spawnargs: [
    '--watch',
    '-d',
    '--progress',
    '--color',
    '--config',
    'webpack/webpack.config.dev.js'
  ]
}

**Comment by geealbers** [comment, marked as answer]
HI @stephaniehanor, sorry for the trouble you're having. I think it may be due to the recent update to the Node LTS version. They moved from 14.18.1 to 16.13.0 last week and I believe a number of Quire's dependencies don't currently work on the newer version. Or at least won't until we can make some updates on our end and publish a new version. In the meantime, can you try the following steps and let us know if it works?

1. Download and install an older version of Node (14.18.1): https://nodejs.org/download/release/v14.18.1/ (If you're on a Mac, you'll want the node-v14.18.1.pkg file)
2. Confirm the Node version by running `node --version` in the command line
3. Delete `themes/default/node_modules` directory and its contents from your project
4. Run `quire install` in your project
5. Run `quire preview --verbose`

If it's still not working, please share the output from the terminal for further trouble shooting and we'll go to plan B. Thanks!

**Comment by stephaniehanor** [comment]
Thanks! I installed the older version and went through these steps. I
received this error message:

mac-974:unseen shanor$ quire preview --verbose

quire preview --verbose

Launching preview server

ℹ Navigate to http://localhost:1313 to see your changes.

ℹ Press Control+C or type "quire stop" to stop the preview.

Start building sites …


Built in 419 ms


Error: Error building site:
"/Users/shanor/unseen/content/catalogue/10.md:27:5": got closing shortcode,
but none is open

Run `quire preview --verbose` for details

Shutting down Quire

mac-974:unseen shanor$

On Mon, Nov 1, 2021 at 11:34 AM Greg Albers ***@***.***>
wrote:

> HI @stephaniehanor <https://github.com/stephaniehanor>, sorry for the
> trouble you're having. I think it may be due to the recent update to the
> Node LTS version. They moved from 14.18.1 to 16.13.0 last week and I
> believe a number of Quire's dependencies don't currently work on the newer
> version. Or at least won't until we can make some updates on our end and
> publish a new version. In the meantime, can you try the following steps and
> let us know if it works?
>
>    1. Download and install an older version of Node (14.18.1):
>    https://nodejs.org/download/release/v14.18.0/ (If you're on a Mac,
>    you'll want the node-v14.18.0.pkg file)
>    2. Confirm the Node version by running node --version in the command
>    line
>    3. Delete themes/default/node_modules directory and its contents from
>    your project
>    4. Run quire install in your project
>    5. Run quire preview --verbose
>
> If it's still not working, please share the output from the terminal for
> further trouble shooting and we'll go to plan B. Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/thegetty/quire/discussions/205#discussioncomment-1571872>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANZDL4FDYRMAF3WDV4N3GLDUJ3MSDANCNFSM5HAJKYRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Dr. Stephanie Hanor
Director
Mills College Art Museum
5000 MacArthur Blvd.
Oakland, CA 94613
619-961-5161 cell
mcam.mills.edu

-- 

Sign up for our MAILING LIST <https://mcam.mills.edu/latest/> and follow us
on FACEBOOK <https://www.facebook.com/MillsArtMuseum> + INSTAGRAM
<https://www.instagram.com/millsartmuseum/>+ TWITTER
<https://twitter.com/MillsArtMuseum>

Read MCAM's archive & collections blog GLASS CUBE
<https://medium.com/glass-cube>

**Reply by geealbers** [reply]
Great!

That new error is unrelated, but if you look at your `catalogue/10.md` Markdown file, on line 27 there's maybe a `{{< /q-class >}}` there that should be removed, or a trailing/extra `>}}`?

**Comment by stephaniehanor** [comment]
Terrific, everything is working! Appreciate it!!



On Mon, Nov 1, 2021 at 12:17 PM Greg Albers ***@***.***>
wrote:

> Great!
>
> That new error is unrelated, but if you look at your catalogue/10.md
> Markdown file, on line 27 there's maybe a {{< /q-class >}} there that
> should be removed, or a trailing/extra >}}?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/thegetty/quire/discussions/205#discussioncomment-1572109>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANZDL4EZAZFSH474XACE3ZLUJ3RVPANCNFSM5HAJKYRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Dr. Stephanie Hanor
Director
Mills College Art Museum
5000 MacArthur Blvd.
Oakland, CA 94613
619-961-5161 cell
mcam.mills.edu

-- 

Sign up for our MAILING LIST <https://mcam.mills.edu/latest/> and follow us
on FACEBOOK <https://www.facebook.com/MillsArtMuseum> + INSTAGRAM
<https://www.instagram.com/millsartmuseum/>+ TWITTER
<https://twitter.com/MillsArtMuseum>

Read MCAM's archive & collections blog GLASS CUBE
<https://medium.com/glass-cube>
